library/src/main/java/dev/chrisbanes/insetter/Insetter.kt

```diff
@@ -536,6 +536,18 @@ class Insetter private constructor(
         fun builder(): Builder = Builder()
 
         internal const val TAG = "Insetter"
+
+        @Deprecated(
+            level = DeprecationLevel.ERROR,
+            message = "Use WindowCompat.setDecorFitsSystemWindows() instead",
+            replaceWith = ReplaceWith(
+                "WindowCompat.setDecorFitsSystemWindows(window, !enabled)",
```

Comment:
I avoided this replacement, as there's no guarantee that `window` will be in the scope

Reply:
Good point!I've removed the replacement statement and left only the function with a comment and ERROR deprecation level.